commercial/app/views/books/booksStandardV2.scala.html

```diff
@@ -2,8 +2,30 @@
   omnitureId: Option[String],
   clickMacro: Option[String])(implicit request: RequestHeader)
 
-@containerWrapper(Seq(), "")(HtmlFormat.empty)(HtmlFormat.empty)(HtmlFormat.empty)(HtmlFormat.empty) {
+@defining(("2_0", "2014-10-15")) { case (version, date) =>
```

Comment:
I'm happy to remove them; it's just that they are used in several places. Well I don't think that's a real problem, let's do that.